src/main/java/com/sysc4806app/model/User.java

```diff
@@ -0,0 +1,20 @@
+package com.sysc4806app.model;
+
+
+public class User {
+
+    private String name;
+
```

Comment:
Should the user's password field be added to the user in this pr or are we gonna add it with the login pr

Reply:
Let's do it with the login PR. I didn't even add an ID or make it an @Entity so that can all happen later